File: pingyougou/pinyougousellergoodsinterface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import entity.PageResult;
import entity.Result;
import com.pinyougou.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {
     List<TbBrand> findAll();
     PageResult findBypage(Integer page, Integer size);
     Result insert(TbBrand tbBrand);
     TbBrand findOne(Long id);
     Result update(TbBrand tbBrand);
     public void delete(Long[] ids);
     PageResult findBypageWithcondition(TbBrand brand, Integer page, Integer size);
     List<Map> findMapList();
}

File: pingyougou/pinyougoumanagerweb/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    this.findAll=function () {
        return $http.get('../brand/find.do')
    }
    this.findBypage=function (page, size) {
        return $http.get('../brand/findBypage.do?page=' + page + '&size=' + size);
    }
    this.insert=function (x) {
        return   $http.post('../brand/add.do', x)
    }
    this.findOne=function (id) {
        return   $http.get('../brand/findOne.do?id='+id)
    }
    this.deleteItem=function (deletListId) {
        return   $http.get("../brand/delete.do?ids="+ deletListId)
    }
    this.search=function (page,rows,query) {
        return  $http.post('../brand/querys.do?page=' + page + '&size=' + rows,query);
    }
    this.findBrandBylist=function () {
        return $http.get('../brand/findBrandBylist.do')
    }
})